collector/lib/CollectorConfig.cpp

```diff
@@ -40,6 +40,8 @@ BoolEnvVar collect_connection_status("ROX_COLLECT_CONNECTION_STATUS", true);
 
 BoolEnvVar enable_external_ips("ROX_ENABLE_EXTERNAL_IPS", false);
 
+BoolEnvVar enable_connection_stats("ROX_COLLECTOR_ENABLE_CONNECTION_STATS", true);
```

Comment:
Shoud this be added to the reference.md page along with the other env vars ? or to the troubleshooting.md (actually I just pushed a small description here) ?

Reply:
Maybe I should rather move those to the reference page....
https://github.com/stackrox/collector/blob/ovalenti/external_networks_metrics/docs/troubleshooting.md#connection-statistics

Reply:
> Shoud this be added to the reference.md page along with the other env vars ? or to the troubleshooting.md

I think both, just under the different context. The first one should contain simply all configurations with minimal explanation what it is for. The second one should describe how to use them for troubleshooting, in this case probably we can simply mention "in case of network load issues, turn this off/on".